SDK/MobileCenter/Microsoft.Azure.Mobile.UWP/Utils/DefaultScreenSizeProvider.cs

```diff
@@ -33,13 +33,13 @@ public DefaultScreenSizeProvider()
                 Task.Factory.StartNew(() =>
                 {
                     var displayInfo = DisplayInformation.GetForCurrentView();
-                    UpdateDisplayInformation(displayInfo, null);
+                    UpdateDisplayInformation((int)displayInfo.ScreenHeightInRawPixels, (int)displayInfo.ScreenWidthInRawPixels);
```

Comment:
But why introducing `UpdateDisplayInformationHandler` ? Does not seem to be used anywhere now.